src/App.tsx

```diff
@@ -40,18 +43,15 @@ function getReadmeFileNames(config: any): string[] {
 	return filenames;
 }
 
-function getExampleFileNames(config: any): string[] {
+function getExampleFileNames(config: Config): string[] {
 	const filenames: string[] = [];
+	if (!config.widgets) return filenames;
 	Object.keys(config.widgets).forEach((key) => {
-		const widget = config.widgets[key];
-		if (widget.overview && widget.overview.example) {
-			filenames.push(config.examplePath(key, widget.overview.example.filename));
-		}
-		if (widget.examples) {
-			widget.examples.forEach((example: any) => {
-				filenames.push(config.examplePath(key, example.filename));
-			});
-		}
+		if (!config.widgets) return;
```

Comment:
would this ever be trie with the if above outside the forEach?

Reply:
No, this was only to satisfy typings.